src/Plugin/facets/query_type/LocalGovDirectoriesQueryType.php

```diff
@@ -87,4 +125,78 @@ public function build() {
     return $this->facet;
   }
 
+  /**
+   * Get facets from a facet group except it's own with a corresponding tag
+   * 
+   * Re-run the search api query, this time removing all the other facet groups
+   * except the one specified by the filter tag. The purpose of this is to get
+   * the other facets that would be limited by setting of the passed in groups
+   * facets. It allows us to see the other possible facets that could be 
+   * selected as part of an 'OR' group 'AND' the facets selected in the passed
+   * in group.
+   *
+   * @param String $filter_tag
+   *   The search api tag of the facet group.
+   * @return Array
+   *   Search api query facet results.
+   */
+  protected function getResultingFacetsFromFacetGroupExceptOwn(String $filter_tag):Array {
+
+    // Set up a special filter query which needs to clone the original.
+    $filter_query = clone $this->query->getOriginalQuery();
+    $filter_query->preExecute();
+
+    // Find conditions
+    $conditions = &$filter_query->getConditionGroup()->getConditions();
+
+    // Store removed conditions so we can reset them.
+    $removed_conditions = [];
+
+    // Loop through each conditions, removing ones that are not this filter tag.
+    foreach ($conditions as $tag => $condition) {
+      if ($condition instanceof ConditionGroupInterface) {
+        $tags = $condition->getTags();
+
+        // @todo Check that we are only removing facet conditions.
+        if (!in_array($filter_tag, $tags)) {
```

Comment:
We could also reverse this proccess, so that we elimnate a specific facet group and find all the other facets that will fit, its a bit of a trial and error and could also lead to the active ones being hidden.